docs/developer-guide/ui-extensions.md

```diff
@@ -57,3 +55,7 @@ Below is an example of a resource tab extension:
 })(window)
 ```
 
+## Application Tab Extensions
+
+The Argo CD Application is a Kubernetes resource so application tabs can be the same way as any other resource tab.
```

Comment:
```suggestion
Since the Argo CD Application is a Kubernetes resource, application tabs can be the same as any other resource tab.
```

small nitpicky grammar correction so it's easier to read

Reply:
thank you!